### PR TITLE
Fix ad9361 linux plaform Makefile

### PR DESCRIPTION
**Pull request by dbogdan**

### CODE REVIEW

File: projects/ad9361/src.mk [hunk 1]
```diff
@@ -20,12 +20,19 @@ SRCS += $(DRIVERS)/axi_core/axi_adc_core/axi_adc_core.c			\
...
-SRCS +=	$(PLATFORM_DRIVERS)/axi_io.c					\
-	$(PLATFORM_DRIVERS)/delay.c
+SRCS +=	$(PLATFORM_DRIVERS)/axi_io.c
+ifeq (linux,$(strip $(PLATFORM)))
+SRCS +=	$(PLATFORM_DRIVERS)/linux_delay.c
+SRCS +=	$(PLATFORM_DRIVERS)/linux_spi.c
+SRCS +=	$(PLATFORM_DRIVERS)/linux_gpio.c
+else
+SRCS +=	$(PLATFORM_DRIVERS)/delay.c
+endif
```
Comment (mchindri) [inline projects/ad9361/src.mk:30]: ds

Comment (mchindri) [inline projects/ad9361/src.mk:30]: Maybe this is simpler. Same for .h files
```suggestion
SRCS +=	$(PLATFORM_DRIVERS)/$(PLATFORM)_spi.c
SRCS +=	$(PLATFORM_DRIVERS)/$(PLATFORM)_gpio.c
```
and: 
```
ifeq (linux,$(strip $(PLATFORM)))
SRCS +=	$(PLATFORM_DRIVERS)/linux_delay.c
else
SRCS +=	$(PLATFORM_DRIVERS)/delay.c
endif
```
Or rename linux_delay.c -> delay.c

Comment (dbogdan) [reply projects/ad9361/src.mk]: Done it for $(PLATFORM)_spi.c and $(PLATFORM)_gpio.c. For consistency, I would better rename delay.c to $(PLATFORM)_delay.c.

**Comment by dbogdan** [comment]
v2:
 - use $(PLATFORM)_spi.c, $(PLATFORM)_gpio.c

**Review by scuciurean — APPROVED** [review]

**Pull request merged**